src/server/test/routes/logsRouteTests.js

```diff
@@ -0,0 +1,302 @@
+const { chai, mocha, expect, app, testUser } = require('../common');
+const moment = require('moment');
+
+mocha.describe('Log Routes', () => {
+   let token;
+
+   mocha.before(async () => {
+       // Login to get authentication token
+       const res = await chai.request(app).post('/api/login')
+           .send({ username: testUser.username, password: testUser.password });
+       token = res.body.token;
+   });
+
+   // Original status code tests
+   mocha.describe('Basic API validation', () => {
+       mocha.it('should return 200 for valid message', async () => {
+           const response = await chai.request(app)
+               .post('/api/logs/info')
+               .set('token', token)
+               .send({ message: 'Valid log message' });
```

Comment:
Maybe for consistency this could be 'Valid info message'. Also similar for the mocha.it above for this test. Similarly for invalid info below.